cfpb/consumerfinance.gov
# OAH: Replace hardcoded icons with standard icon loader

## Changes

- OAH: Replace hardcoded icons with standard icon loader


## How to test this PR

1. Visit http://localhost:8000/owning-a-home/explore-rates/ and see the download icon to top-right of the graph, as well as the right arrows in the "I won't buy for months" tab.


## Screenshots
after/before download icon
<img width="32" alt="Screen Shot 2023-03-17 at 3 36 37 PM" src="https://user-images.githubusercontent.com/704760/226007553-639fb49d-3592-4403-9f4b-f3f9873d0dad.png">

Right arrows
<img width="1160" alt="Screen Shot 2023-03-17 at 3 37 54 PM" src="https://user-images.githubusercontent.com/704760/226007816-dd24fc54-0def-4758-8446-d802bd77bab0.png">
